From e9629fd1cc1047063d502b9edf1069696bb34f96 Mon Sep 17 00:00:00 2001
From: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
Date: Thu, 4 Apr 2024 13:46:12 +0900
Subject: [PATCH 4/4] arm64: dts: renesas: r8a779f0: Fix thermal driver
 register

ECM register was introduced but devicetree was not updated.
This patch fixes it.

Signed-off-by: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a779f0.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a779f0.dtsi b/arch/arm64/boot/dts/renesas/r8a779f0.dtsi
index c543ddbd8f93..3dde00216cc0 100644
--- a/arch/arm64/boot/dts/renesas/r8a779f0.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a779f0.dtsi
@@ -517,7 +517,8 @@ tsc: thermal@e6198000 {
 			reg = <0 0xe6198000 0 0x100>,
 			      <0 0xe61a0000 0 0x100>,
 			      <0 0xe61a8000 0 0x100>,
-			      <0 0xe61b0000 0 0x100>;
+			      <0 0xe61b0000 0 0x100>,
+			      <0 0xe6250000 0 0xa00>;
 			interrupts = <GIC_SPI 768 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&cpg CPG_MOD 919>;
 			power-domains = <&sysc R8A779F0_PD_ALWAYS_ON>;
-- 
2.34.1

